packages/sdk/python/human-protocol-sdk/test/human_protocol_sdk/test_staking.py

```diff
@@ -0,0 +1,290 @@
+import unittest
+
+from eth_typing import URI
+from web3.providers.auto import load_provider_from_uri
+
+from human_protocol_sdk.staking import StakingClient, StakingClientError
+from human_protocol_sdk.utils import get_escrow_interface
+
+from test.human_protocol_sdk.utils import (
+    DEFAULT_GAS_PAYER_PRIV,
+)
+
+
+class StakingTestCase(unittest.TestCase):
+    def setUp(self):
+        self.staking_client = StakingClient(
+            load_provider_from_uri(URI("http://localhost:8545")),
```

Comment:
Let's not use a hardhat provider here. Since it unit testing, let's just mock it.